tests/test_modeling_blenderbot.py

```diff
@@ -13,121 +12,249 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Tests for BlenderBot"""
+""" Testing suite for the PyTorch Blenderbot model. """
+
+
+import tempfile
 import unittest
 
+import timeout_decorator  # noqa
```

Comment:
This doesn't seem to be used

Reply:
Also leaving open as it's in the templates